src/components/AddToPlaylistPopup.js

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+import { addToPlaylist } from "../backend/utils/serviceUtil";
+import { useVideo } from "../context/VideoContext";
+
+export const PlaylistPopup = () => {
+  const { videoDispatch, videoState } = useVideo();
+  const [playlistName, setPlaylistName] = useState("");
+  return (
+    <>
+      <div className="popup-overlay">
+        <div className="popup-container">
+          <div className="popup-header">
+            <span>Save to...</span>
+            <span
+              className="cursor-pointer"
+              onClick={() => videoDispatch({ type: "CLOSE_PLAYLIST_POPUP" })}
+            >
+              <i className="far fa-times"></i>
+            </span>
+          </div>
+          {videoState?.listOfPlaylist.map((i) => (
+            <div className="list-playlist" key={i}>
+              <label>
+                <input type="checkbox" />
+                {i}
+              </label>
+            </div>
+          ))}
+          {/* {videoState.showCreatePlaylistBlock && ( */}
```

Comment:
Remove commented code before merge

Reply:
The logic will be implemented in future, that's why haven't removed the commented code.